driver/random.hpp

```diff
@@ -1,16 +1,28 @@
 #ifndef GUARD_RANDOM_GEN_
 #define GUARD_RANDOM_GEN_
 
-#include <cstdlib>
+#include <random>
+
+std::minstd_rand& get_minstd_gen()
+{
+    static std::random_device rd;
+    static std::minstd_rand minstd_gen(rd());
```

Comment:
`thread_local` for the sake of thread safety.

Reply:
It's better to collapse two static variables into one, especially if the first one is neede just to initialize the second.

Reply:
> It's better to collapse two static variables into one, especially if the first one is needed just to initialize the second.

as in static std::minstd_rand minstd_gen(std::random_device()); ?

Reply:
done :)